src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -101,10 +101,12 @@ const props = defineProps<Props>();
 
 const state = reactive({
     hasManagePermission: useManagePermissionState(),
-    dashboardId: computed<string>(() => props.dashboardId),
-    dashboardViewer: DASHBOARD_VIEWER.PRIVATE as DashboardViewer,
-    dashboardName: 'Dashboard',
-    labelList: [] as Array<string>,
+    dashboardConfig: DASHBOARD_TEMPLATES.monthlyCostSummary, // TODO: should be changed to api data
+    dashboardViewer: computed<DashboardViewer>(() => state.dashboardConfig?.viewer ?? DASHBOARD_VIEWER.PRIVATE),
+    dashboardName: computed<string>(() => state.dashboardConfig?.name ?? ''),
```

Comment:
I think with `computed`, dashboard name edit modal would not work 🥲 
Would you like to check it one more time ?

Reply:
Oh I missed it 🥲 Thanks!!!